Source/DotSpatial.Projections/ProjectionInfo.cs

```diff
@@ -1263,6 +1262,14 @@ public bool TryParseEsriString(string esriString)
             _longitudeOf2nd = GetParameter("Longitude_Of_2nd", esriString);
             LatitudeOfOrigin = GetParameter(new[] { "Latitude_Of_Origin", "Latitude_Of_Center", "Central_Parallel" }, ref _latitudeOfOriginAlias, esriString);
             iStart = esriString.LastIndexOf("UNIT", StringComparison.Ordinal);
+            if (esriString.Contains("VERTCS"))
+            {
+                String woVert = esriString.Substring(iStart);
+                if (woVert.Contains("UNIT"))

```

Comment:
Well of course it contains UNIT if you've started the substring from the word UNIT

Reply:
Also the tag you are after is VERT_CS not VERTCS.
There's also no reason why a VERT_CS cannot come before a PROJCS in a COMPD_CS https://docs.geotools.org/stable/javadocs/org/opengis/referencing/doc-files/WKT.html

Reply:
@pchilds-cordel   So basically merging this fix causes trouble?

Reply:
Neither a fix nor trouble. It is just a No-op.